src/default-retrieve-file-handler.ts

```diff
@@ -1,23 +0,0 @@
-import * as fs from 'fs';
-import { transpileIfTypescript } from './transpile-if-ts';
-
-export function defaultRetrieveFileHandler(path) {
-  // Trim the path to make sure there is no extra whitespace.
-  path = path.trim();
-
-  // This was removed because it seems that we can't use cache while expecting correct results
-  // TODO: check correctness and performance with file caching 
-  // if (path in fileContentsCache) {
-  //   return fileContentsCache[path];
-  // }
-
-  var contents: string;
-  try {
-    contents = fs.readFileSync(path, 'utf8');
-    contents = transpileIfTypescript(path, contents);
```

Comment:
But it seems to me that it is unneeded here, because this function is called when source map is already generated. Actually, I think that we have to force creating of source maps only while transpiling user's ts/tsx code.